docker-compose.yml

```diff
@@ -0,0 +1,34 @@
+version: '3.2'
+services:
+  db:
+    container_name: postgres
+    image: postgres:10.6
+    environment:
+      - PGHOST=postgres
+      - PGUSER=postgres
+      - PGPASSWORD=postgres
+  web:
+    container_name: blockscout
+    build:
+      context: .
+    environment:
+      - PGHOST=postgres
+      - PGUSER=postgres
+      - PGPASSWORD=postgres
+      - ETHEREUM_JSONRPC_VARIANT=geth
+      - COIN=ETH
+      - MIX_ENV=prod
+      - PORT=4000
+      - SECRET_KEY_BASE=RMgI4C1HSkxsEjdhtGMfwAHfyT6CKWXOgzCboJflfSm4jeAlic52io05KB6mqzc5
```

Comment:
Committing secrets?

Reply:
all the time ;) `SECRET_KEY_BASE ` is for [cookie based auth](https://phoenixframework.org/blog/sessions) - the user is suppose to change it themselves before deploying. The one in there is just an example.

Reply:
Good catch though, i'll add a comment or something.